interfaces/apparmor/spec.go

```diff
@@ -190,6 +251,42 @@ func (spec *Specification) setScope(securityTags []string) (restore func()) {
 	}
 }
 
+var metadataTagsSupported = apparmor_sandbox.MetadataTagsSupported
+
+// MetadataTagSnippet wraps the given AppArmor rule snippet in the given
+// metadata tags if tagging is supported, and returns the resulting snippet.
+// If tagging is not supported, returns the snippet unchanged.
+func MetadataTagSnippet(snippet string, tags []MetadataTag) string {
+	if len(tags) == 0 || !metadataTagsSupported() {
+		return snippet
+	}
+
+	var b strings.Builder
+
+	// Put a blank line before the tagged block and open the tags set
+	b.WriteString("\ntags=(")
+
+	// Write the tags, separated by commas
+	for i, tag := range tags {
+		if i > 0 {
+			b.WriteString(",")
```

Comment:
Small nitpick for readability.
```suggestion
			b.WriteString(", ")
```